kpi/signals.py

```diff
@@ -43,38 +43,41 @@ def save_kobocat_user(sender, instance, created, raw, **kwargs):
     grant all KoBoCAT model-level permissions for the content types listed in
     `settings.KOBOCAT_DEFAULT_PERMISSION_CONTENT_TYPES`
     """
-    if not settings.TESTING:
-        KobocatUser.sync(instance)
+    if False:
+        if not settings.TESTING:
+            KobocatUser.sync(instance)
 
-        if created:
-            # FIXME: If this fails, the next attempt results in
-            #   IntegrityError: duplicate key value violates unique constraint
-            #   "auth_user_username_key"
-            # and decorating this function with `transaction.atomic` doesn't
-            # seem to help. We should roll back the KC user creation if
-            # assigning model-level permissions fails
-            grant_kc_model_level_perms(instance)
+            if created:
+                # FIXME: If this fails, the next attempt results in
+                #   IntegrityError: duplicate key value violates unique constraint
+                #   "auth_user_username_key"
+                # and decorating this function with `transaction.atomic` doesn't
+                # seem to help. We should roll back the KC user creation if
+                # assigning model-level permissions fails
+                grant_kc_model_level_perms(instance)
 
 
 @receiver(post_save, sender=Token)
 def save_kobocat_token(sender, instance, **kwargs):
     """
     Sync AuthToken table between KPI and KC
     """
-    if not settings.TESTING:
-        KobocatToken.sync(instance)
+    if False:
+        if not settings.TESTING:
+            KobocatToken.sync(instance)
 
 
 @receiver(post_delete, sender=Token)
 def delete_kobocat_token(sender, instance, **kwargs):
     """
     Delete corresponding record from KC AuthToken table
     """
-    if not settings.TESTING:
-        try:
-            KobocatToken.objects.get(pk=instance.pk).delete()
-        except KobocatToken.DoesNotExist:
-            pass
+    if False:
+        if not settings.TESTING:
+            try:
+                KobocatToken.objects.get(pk=instance.pk).delete()
+            except KobocatToken.DoesNotExist:
+                pass
```

Comment:
Hey @WinnyTroy, looking at this again, can we go with what you had suggested of sticking with `if not settings.TESTING:` and ensuring that `settings.TESTING` has been set to `False`? Let's remove the `if False:` lines that I had earlier suggested. Sorry about this

Reply:
Hi @ivermac Just a reminder that the flag `settings.TESTING` is set to `False` [here](https://github.com/onaio/ansible-kpi/blob/two_db_updates/templates/kpi/settings.py.j2#L227). That hasnt changed even from [kobo's end](https://github.com/kobotoolbox/kpi/blob/master/kobo/settings/base.py#L672)

What I mentioned before was, for us to avoid passing that condition is to change it to `if settings.TESTING:` as that checks if `settings.TESTING` is True